closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java

```diff
@@ -178,6 +200,7 @@ public static NativeCrypto getNativeCrypto() {
         ClassLoader callerClassLoader = Reflection.getCallerClass().getClassLoader();
 
         if ((callerClassLoader == null) || (callerClassLoader == VM.getVMLangAccess().getExtClassLoader())) {
+            initCryptoLibraries();
             return new NativeCrypto();
```

Comment:
This should be a singleton given the new usage pattern.

Reply:
Changed it to a singleton instance.